torchaudio/prototype/models/hdemucs.py

```diff
@@ -937,23 +942,23 @@ def _ispectro(z: torch.Tensor, hop_length: int = 0, length: int = 0, pad: int =
     return x.view(other)
 
 
-def hdemucs_low(sources: List[str], sample_rate: int) -> HDemucs:
+def hdemucs_low(sources: List[str]) -> HDemucs:
     r"""Builds low nfft (1024) version of HDemucs model. This version is suitable for lower sample rates, and bundles
     parameters together to call valid nfft and depth values for a model structured for sample rates around 8 kHZ.
 
     Args:
-        sources (List[str]): Sources to use for audio split
-        sample_rate (int): Serves as metadata, recommend lower sample rates.
+        sources (List[str]): Sources to use for audio split. Input a list of strings out that can contain the following
```

Comment:
To avoid redefining all the definitions, you can also refer users to `HDemucs`, like it is done [here](https://github.com/pytorch/audio/blob/main/torchaudio/models/wav2vec2/model.py#L402). This way there is less repetition and potential future docstring changes only need to be made in one place rather than 4